pkg/chartutil/create.go

```diff
@@ -248,9 +251,18 @@ spec:
         paths:
           {{- range .paths }}
           - path: {{ .path }}
+            {{- if semverCompare "<1.14-0" $capabilities.KubeVersion.GitVersion }}
```

Comment:
If there are multiple locations hardcoding this capability check, we should wrap this in a function call so that the code is more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
I do like the idea of using a function, other than that it all seems ok to me.